flagsmith-core.js

```diff
@@ -281,9 +300,6 @@ const Flagsmith = class {
             return;
```

Comment:
This will be broken now that we have flag analytics. Can't just share an array of unrelated intervals they need to be just separate intervals e.g. this.getFlagInterval and this.analyticsInterval